cookbook/plans/default.json

```diff
@@ -11,16 +11,16 @@
                     "denom": "ulava",
                     "amount": "100000"
                 },
-                "annual_discount_percentage": "20",
+                "annual_discount_percentage": 20,
                 "allow_overuse": false,
-                "overuse_rate": "0",
+                "overuse_rate": 0,
                 "plan_policy": {
                     "chain_policies": [
                     ],
-                    "geolocation_profile": "18446744073709551615",

```

Comment:
needs to be global

Reply:
done. see https://github.com/lavanet/lava/commit/2c666da79a87eeff3e849112b428e2549a828f8f